### ISSUE TITLE
Improve model

### ISSUE THREAD

**Issue by alif-munim** [alif-munim/r-tensorflow#2]
Plot loss and accuracy. Improve neural network and prevent overfitting by including more hidden and dropout layers.
